windows/jitsi-meet/src/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import {
+    RemoteControl,
+    setupScreenSharingForWindow,
+    setupAlwaysOnTopRender,
+    setupWiFiStats
+} from 'jitsi-meet-electron-utils';
+
+import { jitsiMeetDomain } from '../../../config.js';
+
+/**
+ * Jitsi Meet Window Component
+ */
+class JitsiMeetWindow extends React.Component {
+
+    /**
+     * Render function of component
+     * @return {ScriptElement}
+     */
+    render() {
+        return null;
+    }
+
+    /**
+     * Attach the script
+     */
+    componentDidMount() {
+
+        const script = document.createElement('script');
+
+        script.async = true;
+        script.onload = this.onScriptLoad;
+        script.onerror = console.error;
+        script.src = `https://${jitsiMeetDomain}/external_api.js`;
+
+        document.head.appendChild(script);
+    }
+
+    /**
+     * When the script is loaded attach utils from jitsi-meet-electron-utils
+     */
+    onScriptLoad() {
```

Comment:
Make this method "private", prefix it with `_`, that is.

Reply:
We don't follow that style guide. We follow our own :-) See the jitsi-meet project, we use it extensively.

Reply:
So yeah, please change it.

Reply:
Yeah. I know we don't follow that guide. But their reason seems good.
Adding "_" as the prefix neither does anything nor it is a convention. 

Anyways, I have updated the PR.